src/screens/SearchScreen.tsx

```diff
@@ -1,19 +1,31 @@
-import React, { useState } from 'react'
-import { StyleSheet, View } from 'react-native'
+import React, { useRef, useState } from 'react'
+import { StyleSheet, View, TextInput } from 'react-native'
+import AntIcon from 'react-native-vector-icons/AntDesign'
+import EntypoIcon from 'react-native-vector-icons/Entypo'
+import { TouchableOpacity } from 'react-native-gesture-handler'
 
-import SearchBar from '../components/Search'
+import { SearchBar } from '../components/Search'
```

Comment:
Why not make `SearchBar` a component that lives outside of the `Search` folder?

Reply:
Well earlier I had `SearchFilter` component here too which now is deprecated. I decided to keep it this way since I will get `SearchResults` component here in next PR

Reply:
Ok. Keep it here for now, but I think we'll restructure to either colocating components with their presenters, or keeping it as flat as we can in components folder